main.go

```diff
@@ -297,11 +299,19 @@ func initDaemon(config *menderConfig, dev *device, env *uBootEnv,
 
 	authmgr := NewAuthManager(store, config.DeviceKey, NewIdentityDataGetter())
 
+	//configure deployment logging
+	deploymentLogsLocation := config.GetDeploymentLogLocation()
+	if deploymentLogsLocation == "" {
+		deploymentLogsLocation = defaultDeploymentLogLocation
+	}
+	logManager := NewDeploymentLogManager(deploymentLogsLocation)
+
 	controller, err := NewMender(*config, MenderPieces{
 		device:  dev,
 		env:     env,
 		store:   store,
 		authMgr: authmgr,
+		logMgr:  logManager,
```

Comment:
Don't think log manager should be part of mender. IMO it's outside of controller's area of 'control'